optking/stepAlgorithms.py

```diff
@@ -12,6 +12,10 @@
     * SteepestDescent
         * overlap
         * barzilai_borwein
+    * ConjugateGradient
```

Comment:
Since these are not very common options, could you include a very short reference (e.g., Fletcher, Ch. 2, Eqn xx) or something like that?